src/drivers/codeModel.ts

```diff
@@ -60,7 +60,7 @@ export interface CodeModelFileGroup {
     }[];
 
     /** Compiler flags */
-    compileFlags?: string;
```

Comment:
I think we need to keep both flags and fragments and populate the right one based on the cpptools api.